src/python/pants/rules/core/test.py

```diff
@@ -137,7 +137,7 @@ class TestOptions(GoalSubsystem):
 
     name = "test"
 
-    required_union_implementations = (TestTarget, CoverageDataBatch)
+    required_union_implementations = (TestTarget,)
```

Comment:
@TansyArron my bad for originally recommending that you do this.

FWIT, I recommend getting in the habit of running `./v2 test path/to/file_test.py` so that you get dogfooding of `test`. (Also, precise file args save a bunch of time and only work in V2).

Reply:
👍 I did run the test_test.py tests, but I ran them on a branch that also had the pytest changes on it. 

...So this passed CI because none of the tests actually ran? That's frightening.